app.gummi.gummi.yml

```diff
@@ -0,0 +1,90 @@
+id: app.gummi.gummi
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+command: gummi
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.texlive
+
+add-extensions:
+  org.freedesktop.Sdk.Extension.texlive:
```

Comment:
either you use add-extensions or you use the install script. not both.

Reply:
what this means is that choice number 2 involve remove the `add-extensions` altogether.

Reply:
I removed the sdk-extension, I won't want to use the extension installed in my application